src/helpers/openAppTokenLink.ts

```diff
@@ -0,0 +1,4 @@
+export default function ({ blank, token }: { token: string; blank?: boolean }) {
```

Comment:
I think we can modify existing helper instead of creating a new one

Reply:
Also, re-use `TokenScreenParams` for props here, when you make `token` non-optional